src/components/layout/footer.module.scss

```diff
@@ -1,99 +1,143 @@
-.footer {
-  background-size: auto 300%;
-  background-color: $color-plum-550;
-  color: white;
-  margin-top: spacer(32);
-  padding: spacer(32) 0;
-
-  @media (max-width: $viewport-ms) {
-    .footer-list {
-      flex-direction: column;
-    }
-  }
+.wrapper {
+  background-color: $color-plum-700;
+}
 
-  @media (min-width: $viewport-ms) {
-    .container {
-      align-items: center;
-      display: flex;
-      justify-content: space-between;
-    }
-  }
+.footer,
+.copyright {
+  width: 71.25rem;
+  max-width: calc(100vw - 4rem);
+  margin: 0 auto;
 
-  img {
-    width: toRem(176);
-    margin: 0;
-    margin-right: 1rem;
+  a,
+  span {
+    color: white;
   }
+
   a {
+    font-weight: bold;
+    text-decoration: none;
+
     &:focus {
       @include link-focus-dark-bg;
     }
     &:hover {
       @include link-hover-dark-bg;
     }
   }
-  .footer-list {
+}
+
+.footer {
+  @include padding(8, bottom);
+  @include padding(32, top);
+
+  img {
+    width: 32px;
+  }
+
+  .project-logo {
+    width: toRem(176);
+    margin: 0;
+    @include margin(16, right);
+  }
+}
+
+.main {
+  @include row();
+  align-items: center;
+
+  ul {
     list-style-type: none;
-    padding: 0;
-    display: flex;
-    flex-wrap: wrap;
-    justify-content: right;
-    @media (min-width: $viewport-md) {
-      margin: 0;
+    @include padding(0, left);
+  }
+
+  .ctp-logo {
+    @include col(3 3 3);
+
+    @media (max-width: $viewport-lg) {
+      order: 1;
     }
+  }
 
-    @media (min-width: $viewport-sm) {
-      li {
-        display: inline-block;
-        margin: spacer(4) 0 spacer(4) spacer(24);
-        margin-bottom: 0;
-        &:last-of-type {
-          margin-right: 0;
-        }
-        &:first-of-type {
-          margin-left: 0;
-        }
+  .links {
+    @include col(4 6 7);
+    ul {
+      display: grid;
+      grid-template-columns: repeat(3, 1fr);
+      grid-template-rows: repeat(2, 1fr);
+      grid-auto-flow: column;
+      @media (max-width: $viewport-lg) and (min-width: $viewport-md) {
+        grid-template-columns: repeat(2, 1fr);
+        grid-template-rows: repeat(3, 1fr);
+      }
+      @media (max-width: $viewport-md) {
+        grid-template-columns: 1fr;
+        grid-template-rows: repeat(6, 1fr);
       }
     }
 
-    @media (max-width: $viewport-ms) and (min-width: $viewport-sm) {
-      li:first-of-type {
-        margin-left: 1.5rem;
-      }
+    @media (max-width: $viewport-lg) {
+      order: 3;
     }
+  }
 
-    a {
-      color: white;
-      font-weight: bold;
-      text-decoration: none;
-      &:hover {
-        @include link-hover-dark-bg;
-      }
+  .social {
+    @include col(1 3 2);
+    @include margin(0, (top, bottom));
+
+    display: flex;
+    justify-content: right;
+
```

Comment:
Do we want the social icons to float right on the medium breakpoint?
![image](https://user-images.githubusercontent.com/270536/85185663-32f1ef00-b252-11ea-9c1f-8b86b61417ab.png)

Reply:
I believe so, and I think they `float: right;` now. Do you want to change that behavior?

Reply:
Hm, this is what I'm getting on Chrome:

![image](https://user-images.githubusercontent.com/270536/85203631-4b065480-b2cc-11ea-8cae-9b3bb1f33458.png)

Reply:
Fixed in 23cf28d0b3dba16f8f52f0994cdb412d44c1f7f0 -- see the commit message for details